app/cli/cmd/attached_integration_list.go

```diff
@@ -53,22 +53,32 @@ func attachedIntegrationListTableOutput(attachments []*action.AttachedIntegratio
 	fmt.Println("Integrations attached to workflows")
 	t := newTableWriter()
 	t.AppendHeader(table.Row{"ID", "Kind", "Config", "Attached At", "Workflow"})
-	for _, i := range attachments {
-		wf := i.Workflow
-		integration := i.Integration
+	for _, attachment := range attachments {
+		wf := attachment.Workflow
+		integration := attachment.Integration
+
+		// Merge attachment and integration configs
+		// If there are not attachment configuration then create an empty map
+		if attachment.Config == nil {
```

Comment:
I noticed that this code was buggy when the registration didn't have any metadata.